codalab/client/local_bundle_client.py

```diff
@@ -772,9 +776,8 @@ def ensure_unused_worksheet_name(self, name):
         username = self._current_user_name()
         if spec_util.is_home_worksheet(name) and spec_util.home_worksheet(username) != name:
             raise UsageError('Cannot create %s because this is potentially the home worksheet of another user' % name)
-
         try:
-            self.get_worksheet_uuid(None, name)
+            self.ensure_worksheet_exist(None, name)
```

Comment:
That's actually my intention. Sorry for the bad naming. 

ensure_worksheet_exist will throw an error if the worksheet doesn't exist 
It won't create a new one.

Maybe I should rename it to something else?